### PR TITLE
Extend without object

### PR DESCRIPTION
**Pull request by caseywatts**
This should fix the issue found in:
https://github.com/netguru/eslint-plugin-netguru-ember/issues/27

`TypeError: Cannot read property 'properties' of undefined`

### CODE REVIEW

File: rules/utils/ember.js [hunk 1]
```diff
@@ -267,7 +267,12 @@ function isControllerDefaultProp(property) {
 }
 
 function getModuleProperties(module) {
-  return utils.findNodes(module.arguments, 'ObjectExpression')[0].properties;
+  var firstObjectExpressionNode = utils.findNodes(module.arguments, 'ObjectExpression')[0];
+  if (firstObjectExpressionNode) {
+    return firstObjectExpressionNode.properties;
+  } else {
+    return [];
+  }
```
Comment (azranel) [inline rules/utils/ember.js]: How about
```
return firstObjectExpressionNode ? firstObjectExpressionNode.properties : [];
```

I think this way it's more compact and readability doesn't really suffer 😄

Comment (caseywatts) [inline rules/utils/ember.js]: great idea :)
done!

**Comment by michalsnik** [comment]
Hey @caseywatts. Thanks for tracking this down and providing a solution.

However I'm thinking about one thing - those tests are not actually relevant with their corresponding rules. And I think it would be better to improve test for this utility function. Unfortunately my PR #24  is yet to be checked, so anyway I'm willing to merge this and eventually work on this later on :)

Thanks for your contribution!

**Comment by caseywatts** [comment]
@michalsnik ooh agreed, utils tests would be a better place for testing this functionality.

Relatedly/interestingly/worthnoting - when `alias-model-in-controller` is enabled, then `Controller.extend()` isn't valid - that's the one test that didn't get that test line added in this PR (because it wouldn't have been valid).

**Comment by michalsnik** [comment]
I think we're good. This additional test is rather not necessary. I'll work on testing ember utilities and I'll add your test in proper place finally :) Unless you want and have time to help and write some tests for ember utilities - then be my guest (tests for other utilities are already on master). I would really appreciate any help :)

**Pull request merged**